_markets/de/buy-resell/products/_archive.mdx

```diff
@@ -1,11 +1,22 @@
-### POS Archiv
+We offer two solutions for archiving data, see their Product Features below and [find more product details here](https://fiskaltrust.de/produkte/).
+MeinFiskal is described with its [technical details here](https://docs.fiskaltrust.cloud/de/docs/posdealers/buy-resell/products/3rd-party/datev-meinfiskal).
 
-The POS Archive is a **cloud service** which provides a searchable, secure, and **audit-proof** database for Middleware receipts. Activating it for a CashBox will also enable its Middleware instance to regularly push its new receipts into the archive, where they'll be securely persisted in a revision-safe storage. Due to german regulations, the data is kept for **at least ten years**.
+| Product Features | DATEV MeinFiskal    | PosArchiv           |
+|:----------------------|:--------------------|:----------------------|
+|Included in the Carefree Bundle |Yes  |Yes
+|Available as single product |No (in progress)  |Yes
+|Setup for multiple PosOperators |No |Yes  |
+|Place of backup |Germany  |Germany
+|Duration of backup |ten years  |ten years
+|Backup content |DFKA-Files (Daily-Closing), Receipts sent by fiskaltrust.Middleware and fiskaly relevant documentation   |Receipts sent by fiskaltrust.Middleware
+|Necessary requirements |Master Data complete and validated. CashBox active and running. No Account existing on MeinFiskal with similar E-mail-Adress, Daily-Closing receipt required  |Entitlement needed, Daily-Closing receipt required
+|Third party involved |Integration of DATEV MeinFiskal to the fiskaltrust.Portal  |No
+|Searchable backup |Yes  |Yes
+|GoBD - Conformity: |Yes  |Yes
+|Export options: full Export |Export of all data sent to the queue of a CashBox in ~.xml or ~.csv format  |Export of all data sent to the queue of a CashBox in ~.xml or ~.csv format
+|Export options: log data of TSE |Export of the log messages generated by the TSE in ~.tar format |Export of the log messages generated by the TSE in ~.tar format
+|Export options: DSFinV-K |Yes  |Yes
+|Export options: DFKA |Yes|Not available without the Carefree Bundle  |
+|Option for inspection or further processing |PosOperator can access data during tax audits via DATEV Kassenarchiv online and fiskaltrust.Portal.PosDealer cannot access data. **Forwarding** (with costs) to **DATEV Kassenbuch online for accounting or tax consulting**.  |PosOperator can access data during tax audits via fiskaltrust.Portal. PosDealer can also access data.
+|Use without fiskaltrust.Middleware |(Manual) upload of middleware data from any Pos-System possible  |No
```

Comment:
Let's omit this, IMO it's more confusing than helpful :) 
The whole point of all this things is that you don't need to do anything manually, this sounds like you maybe should